e2e/spec/functional/behavioral-popup.spec.js

```diff
@@ -96,7 +96,7 @@ describe('Open: load (via embed code)', () => {
 
   it('should display close button', () => {
     cy.get('.typeform-share > .icon > img[data-qa="popup-close-button"]').should('be.visible')
-    cy.get('.typeform-share > .icon > svg').should('not.be.visible')
+    cy.get('.typeform-share > .icon > svg').should('not.exist')
```

Comment:
Change for updated Cypress. Element does not exist. In previous version it worked, now it throws an error since it is not "not visible" but it does "not exit".